lib/image-diff.js

```diff
@@ -2,16 +2,17 @@
 var fs = require('fs');
 var path = require('path');
 var async = require('async');
-var gm = require('gm').subClass({imageMagick: true});
+var gm;
 var bufferedSpawn = require('buffered-spawn');
 var mkdirp = require('mkdirp');
 var tmp = require('tmp');
 
 // Define custom resize function
-// Taken from https://github.com/twolfson/twolfson.com/blob/3.4.0/test/perceptual-tests/twolfson.com_test.js#L88-L107
-// TODO: Make image resizing its own library
-// DEV: This does not pollute gm.prototype
-gm.prototype.fillFromTo = function (params) {
+function defineFillFromTo() {
+  // Taken from https://github.com/twolfson/twolfson.com/blob/3.4.0/test/perceptual-tests/twolfson.com_test.js#L88-L107
+  // TODO: Make image resizing its own library
+  // DEV: This does not pollute gm.prototype
+  gm.prototype.fillFromTo = function (params) {
```

Comment:
Unfortunately, this does pollute the prototype now that we can be passing in `gm` directly without `imageMagick` subclassing. As a result, we should now make this a vanilla function:

```js
function fillFromTo(gm, params) {
  gm.borderColor('transparent');
  // ...
  return gm
}

// When used
fillFromTo(this.gm(actualPath), {
  fromWidth: actualSize.width,
  fromHeight: actualSize.height,
  toWidth: maxWidth,
  toHeight: maxHeight
})
```